src/EnergyPlus/UnitarySystem.cc

```diff
@@ -5174,17 +5260,22 @@ namespace UnitarySystems {
                 errorsFound = true;
             }
 
-            if (this->m_CoolingCoilType_Num == DataHVACGlobals::CoilDX_MultiSpeedCooling) {
-                this->m_DiscreteSpeedCoolingCoil = true;
-            } else if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingWaterToAirHPVSEquationFit ||
-                       this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed) {
-                this->m_ContSpeedCoolingCoil = true;
-            } // CoilDX_Cooling is set above
-
-            if (SetPointManager::NodeHasSPMCtrlVarType(state, this->AirOutNode, SetPointManager::iCtrlVarType::Temp))
-                this->m_SystemCoolControlNodeNum = this->AirOutNode;
-            if (SetPointManager::NodeHasSPMCtrlVarType(state, CoolingCoilOutletNode, SetPointManager::iCtrlVarType::Temp))
-                this->m_SystemCoolControlNodeNum = CoolingCoilOutletNode;
+            if (loc_m_SystemCoolControlNodeName.size() > 0) {
```

Comment:
CoilSystem allows control node to be other than outlet node which is same as coil outlet node. One example file was not running since the control node wasn't known.